02_configure_host.sh

```diff
@@ -21,7 +21,7 @@ if [[ "${IPA_DOWNLOAD_ENABLED}" = "true" ]] || [[ ! -r "${IRONIC_DATA_DIR}/html/
         sudo "${CONTAINER_RUNTIME}" run --rm --net host --name ipa-downloader ${POD_NAME} \
           -e "IPA_BASEURI=${IPA_BASEURI:-}" \
           -v "${IRONIC_DATA_DIR}:/shared" "${IPA_DOWNLOADER_IMAGE}" \
-          /bin/bash -c "/usr/local/bin/get-resource.sh &> /dev/null" && s=0 && break || s=$?
+          /bin/bash -c "http_proxy=${http_proxy:-} https_proxy=${https_proxy:-} /usr/local/bin/get-resource.sh &> /dev/null" && s=0 && break || s=$?
```

Comment:
In case the variables contain space, this will break the syntax. Add quotes around the variables, and mind that the command is already once quoted.

Reply:
What does this command even do? It seems to just set the variables to themselves. What I think is needed instead, is something like `-e HTTP_PROXY` passed to podman/docker (and needs handling HTTP_PROXY vs http_proxy).